app/controllers/bookingsync_portal/admin/connections_controller.rb

```diff
@@ -6,7 +6,12 @@ def create
           new_remote_rental = BookingsyncPortal.remote_rental_model.constantize.new(remote_account: remote_account)
           @connection = rental.create_connection(remote_rental: new_remote_rental)
         else
-          @connection = rental.create_connection(remote_rental: remote_rental)
+          @connection =
```

Comment:
how does it work here actually? I mean UI-wise. Is customer allowed to disconnect rental and connect another one, doesn't look like that? Do we want to protect customer from him? This might be better but I can see plenty of support tickets to destroy connection because of some reason. 
Also how would you reconnect it/restore it? Is UI ready for that? I mean we have disconnect button which will cancel it, but connection is still present so probably rental will not appear to be draggable. If not draggable it's not possible to connect this rental with another remote rental which happens sometimes (or even quite often in case of airbnb)

Reply:
Here is how this will work, a new connection is created on a new remote rental connection, for existing remote rentals that have their connection already destroy, we also create a new connection,  for remote rentals with canceled connections we simply restore.
If a customer would like to destroy a connection, we can handle that via console.

Reply:
Hmm all right so if I imagine it correctly it will act in UI as disconnected. What happens if someone drags and drops new rental on remote rental that was connected to other rental? 

Handling destroy via console sounds really painful to me. I see that we want to protect customers from themselves, but this looks like unending pings from support to do that, which is not ideal as well.